src/InfluxDB/Client.php

```diff
@@ -235,37 +222,33 @@ public function listUsers()
 
     /**
      * Build the client from a dsn
-     *
      * Example: https+influxdb://username:pass@localhost:8086/databasename', timeout=5
      *
-     * @param string $dsn
-     *
-     * @param int    $timeout
-     * @param bool   $verifySSL
-     *
+     * @param  string $dsn
+     * @param  int    $timeout
+     * @param  bool   $verifySSL
      * @return Client|Database
-     *
      * @throws ClientException
      */
     public static function fromDSN($dsn, $timeout = 0, $verifySSL = false)
     {
         $connParams = parse_url($dsn);
         $schemeInfo = explode('+', $connParams['scheme']);
-        $dbName = null;
-        $modifier = null;
-        $scheme = $schemeInfo[0];
+        $dbName     = null;
+        $modifier   = null;
```

Comment:
This change is not needed, nor is it in line with our coding standards. Sorry :(